src/graphics/aurora/fadequad.h

```diff
@@ -0,0 +1,77 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  The Quad to handle one color fades.
+ */
+
+#ifndef XOREOS_FADEQUAD_H
+#define XOREOS_FADEQUAD_H
+
+#include "src/graphics/renderable.h"
+
+namespace Graphics {
+
+namespace Aurora {
+
+class FadeQuad : public Renderable {
+public:
+	FadeQuad();
+
+	/** Get the current color of the quad */
```

Comment:
There needs to be a period at the end of the sentence here and at the end of all the other doxygen comments.